app-frontend/src/app/pages/projects/edit/annotate/annotate.controller.js

```diff
@@ -23,6 +25,8 @@ export default class AnnotateController {
         this.isUpload = true;
         this.bindUploadEvent();
 
+        this.clickedId = '';
```

Comment:
No reason to define this here - if we use '' as the default, non selected id, then we can't just do `if(this.clickedId)` to see if something's been clicked. Likewise, if we want to not have a clickedId, use `delete this.clickedId`